package.json

```diff
@@ -14,6 +14,7 @@
     "lodash": "^4.6.1",
     "strformat": "^0.0.7",
     "uber-licence": "^2.0.1",
+    "underscore": "^1.5.2",
```

Comment:
It feels strange to have both underscore and lodash in dependencies. Can we get rid of one of them?